ethereum/contracts/BasicOutboundChannel.sol

```diff
@@ -17,7 +17,8 @@ contract BasicOutboundChannel is OutboundChannel, ChannelAccess, AccessControl {
         address source,
         address account,
         uint64 nonce,
-        bytes payload
+        bytes payload,
+        uint64 weight
```

Comment:
I would swap the order of these fields so that the payload is last.

Reply:
Swapped in eff0bb0d